playbooks/roles/buildbot_master/tasks/main.yml

```diff
@@ -0,0 +1,90 @@
+---
+#
+# edX Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://github.com/edx/configuration/wiki
+# code style: https://github.com/edx/configuration/wiki/Ansible-Coding-Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+#
+#
+# Tasks for role buildbot_master
+#
+# Overview:
+#
+#
+# Dependencies:
+#   - common
+#
+#
+# Example play:
+#
+#
+
+- name: Create buildbot_master group
+  group:
+    name: "{{ buildbot_master_group }}"
+    state: present
+
+- name: Add the buildbot_master user to the group
+  user:
+    name: "{{ buildbot_master_user }}"
+    append: yes
+    groups: "{{ buildbot_master_group }}"
+
+- name: Make buildbot_master directory
+  file:
+    path: "{{ buildbot_master_dir }}"
+    state: directory
+    owner: "{{ buildbot_master_user }}"
+    group: "{{ buildbot_master_group }}"
+    mode: 0755
+
+# TODO: Create task for stopping master if master exists
+# Currently, you can only reprovision while master is not running. To
+# reprovision, first either manually stop master or run 'vagrant halt'.
+
+- name: Create virtualenv for buildbot_master
+  command: virtualenv --clear "{{ buildbot_master_venv }}"
+  become: yes
+  become_user: "{{ buildbot_master_user }}"
+
+- name: Install txrequests
+  pip:
+    name: txrequests
+    virtualenv: "{{ buildbot_master_venv }}"
+  become: yes
```

Comment:
Look into if this is the usual way of specifying user for every task instead of somewhere once

Reply:
It looks like other roles do something similar (now deprecated). This appears to be the preferred method in Ansible 1.9.